src/entities/virgin.entity.ts

```diff
@@ -69,6 +70,10 @@ export class VirginEntity extends BaseEntity {
   @ManyToOne({ nullable: true })
   intro_song?: IntroSongEntity;
 
+  // For checking last time a into was played
+  @Property({ type: DateTimeType, nullable: true })
```

Comment:
Add a doc comment. Maybe "A timestamp for the last time an intro song was played for this user."